src/widgets/pagination/__tests__/pagination-test.js

```diff
@@ -73,7 +73,7 @@ describe('pagination()', () => {
   });
 
   it('sets the page', () => {
-    widget.refine(helper, 42);
+    widget.refine(42);
```

Comment:
was this a bug that was there all this time?
There is another call to refine to fix on [line 104](https://github.com/algolia/instantsearch.js/pull/4199/files#diff-d23004038675fb0d352846b17da97790L104)

Reply:
this was a bug in the test, yes. I'll fix the other occurrance too.

Reply:
After fully reviewing, this test doesn't make sense here in the end. See https://github.com/algolia/instantsearch.js/pull/4199#discussion_r349163769.